mllib-local/pom.xml

```diff
@@ -75,6 +75,11 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>dev.ludovic.netlib</groupId>
+      <artifactId>blas</artifactId>
```

Comment:
So graphx and mllib-local would not need ARPACK, LAPACK?

Reply:
There is currently no uses for ARPACK nor LAPACK in mllib-local or graphx. The only place it's used is in `org.apache.spark.mllib.linalg.CholeskyDecomposition` and `org.apache.spark.mllib.linalg.EigenValueDecomposition`.